tsl/src/data_node.c

```diff
@@ -635,7 +642,8 @@ data_node_validate_extension_availability(TSConnection *conn)
 
 	if (PQresultStatus(res) != PGRES_TUPLES_OK)
 		ereport(ERROR,
-				(errcode(ERRCODE_CONNECTION_EXCEPTION), errmsg("%s", PQresultErrorMessage(res))));
+				(errcode(ERRCODE_CONNECTION_EXCEPTION),
```

Comment:
Why not use `remote_result_elog` here as well?

Reply:
`remote_result_elog` does not allow adding a custom error message, like done here (with the change).